src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php

```diff
@@ -53,6 +54,10 @@ class DebugHandlersListener implements EventSubscriberInterface
      */
     public function __construct(callable $exceptionHandler = null, LoggerInterface $logger = null, $levels = \E_ALL, ?int $throwAt = \E_ALL, bool $scream = true, $fileLinkFormat = null, bool $scope = true)
     {
+        $handler = set_exception_handler('var_dump');
```

Comment:
The same 3 lines of code are also in the configure function (line 83-85).  Should these be move to new private function? e.g.
```
private function setErrorHandlerToVarDump()
{
    $handler = set_exception_handler('var_dump');
    $handler = \is_array($handler) ? $handler[0] : null;
    restore_exception_handler();

    return $handler;
}
```
Constructur can then be:
```
$this->earlyHandler = $this->setErrorHandlerToVarDump();
```

And configure function:
```
$handler = $this->setErrorHandlerToVarDump();
```

Reply:
not worth it imho, duplication doesn't always deserve factorization :)
but thanks for the suggestion.